source/includes/_webhooks.md

```diff
@@ -24,29 +24,63 @@ It is advisable to have tolerance for delivering a callback maximum up to ten mi
 
 Our implementation follows [Stripe's specification](https://stripe.com/docs/webhooks/signatures)
 
-## Incidents
+`dont_disturb_mode` user setting in mobile application has no influence on these events.
 
-This is a definition of an incident payload sent via webhook.
+## Incident events
 
-### parameters
+Webhook will receive all events related to incidents.
+
+Incidents have the following lifecycle:
+
+```
+created -> dismissed -> resolved
+      │                 ▲
+      └─────────────────┘
+```
+
+### Parameters
 
 Parameter | Description
 --------- | -----------
-type | Type of the webhook's data. It will be `incident`.
+type | A string describing the incident event type
```

Comment:
This should not be changed. I have not described "generic" webhook format as we don't have anything except incident however my initial idea was to have a generic format like:
```
{
   "type":"webhook type"
   "data":"any data relevant for the type"
}
```

The type here must be `incident` here. Later on this will allow us to introduce different webhooks. 

Since you are describing state of the incident I would suggest to call it `state`. However this must be again in `data` part.

Reply:
I realized that it will be a generic webhook type when I saw the format. That's why I prepended `created`, `dismissed`, etc with `incident_` getting `incident_created`, `incident_dismissed`, etc.
 
So we can have other types like: `space_created`, `space_updated`, `space_deleted`, `property_changed`, `membership_requested`, `remote_restart_started`, `remote_restart_completed`, `remote_restart_failed`, etc

You not always can infer the `action` you have done other a model from its current state, for example `reopen` action over the `incident` will put it in the `:contacting` state, and you have no idea if it was `create`, `escalate` or `reopen` action looking only on its state.

Reply:
Ok, I don't have strong preferences although I would not change it. If you wish to keep it on top level use dot notation please to fully distinguish between type of event and action e.g `incident.created`

Reply:
Good, I'll use the dot notation, thanks. Addressed